synth.py

```diff
@@ -10,18 +10,16 @@
 
 # tasks has two product names, and a poorly named artman yaml
 for version in ['v1', 'v1beta2']:
-    library = gapic.node_library(
-        'language', version)
+    library = gapic.node_library('language', version)
 
     # skip index, protos, package.json, and README.md
-    s.copy(
-        library,
-        excludes=['package.json', 'README.md', 'src/index.js'])
+    s.copy(library, excludes=['package.json', 'README.md', 'src/index.js'])
 
-templates = common_templates.node_library(package_name="@google-cloud/language")
+templates = common_templates.node_library(
+    package_name="@google-cloud/language",
+    repo_name="googleapis/nodejs-language")
```

Comment:
Before we go adding this stuff everywhere....  gosh this feels like data that synthtool could read from package.json.